run/test/specs/message_checks_groups_android.spec.ts

```diff
@@ -0,0 +1,546 @@
+  await device3.clickOnElementAll({
+    strategy: "id",
+    selector: `network.loki.messenger:id/scrollToBottomButton`,
+  });
+  await device3.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: replyMessage,
+  });
+
+  // Close server and devices
+  await closeApp(device1, device2, device3);
+}
+
+async function sendVideoGroup(platform: SupportedPlatformsType) {
+  // Test sending a video
+  // open devices
+  const testGroupName = "Message checks for groups";
+  const { device1, device2, device3 } = await openAppThreeDevices(platform);
+  // Create users A, B and C
+  const [userA, userB, userC] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+    newUser(device3, "Charlie", platform),
+  ]);
+  // Create contact between User A and User B
+  await createGroup(
+    platform,
+    device1,
+    userA,
+    device2,
+    userB,
+    device3,
+    userC,
+    testGroupName
+  );
+  const replyMessage = `Replying to video from ${userA.userName} in ${testGroupName}`;
+  // Click on attachments button
+  await device1.clickOnElement("Attachments button");
+  await sleepFor(100);
+  // Select images button/tab
+  await device1.clickOnElement("Documents folder");
+  // Select video
+  const mediaButtons = await device1.findElementsByClass(
+    "android.widget.CompoundButton"
+  );
+  const videosButton = await device1.findMatchingTextInElementArray(
+    mediaButtons,
+    "Videos"
+  );
+  if (!videosButton) {
+    throw new Error("videosButton was not found");
+  }
+  await device1.click(videosButton.ELEMENT);
+  const testVideo = await device1.doesElementExist({
+    strategy: "id",
+    selector: "android:id/title",
+    maxWait: 1000,
+    text: "test_video.mp4",
+  });
+  if (!testVideo) {
+    // Adds video to downloads folder if it isn't already there
+    await runScriptAndLog(
+      `adb -s emulator-5554 push 'run/test/specs/media/test_video.mp4' /storage/emulated/0/Download`,
+      true
+    );
+  }
+  await sleepFor(100);
+  await device1.clickOnTextElementById("android:id/title", "test_video.mp4");
+  // Reply to message
+  await device2.waitForTextElementToBePresent({
+    strategy: "id",
+    selector: "network.loki.messenger:id/play_overlay",
+    maxWait: 8000,
+  });
+  await device3.waitForTextElementToBePresent({
+    strategy: "id",
+    selector: "network.loki.messenger:id/play_overlay",
+    maxWait: 8000,
+  });
+
+  await device2.longPress("Media message");
+  await device2.clickOnElement("Reply to message");
+  await device2.sendMessage(replyMessage);
+  await sleepFor(2000);
+  await device1.clickOnElementAll({
+    strategy: "id",
+    selector: `network.loki.messenger:id/scrollToBottomButton`,
+  });
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: replyMessage,
+  });
+  await device3.clickOnElementAll({
+    strategy: "id",
```

Comment:
lots of `strategy: "id"` in those tests too

Reply:
is that bad? @Bilb